### PR TITLE
Vxpolls improved exporting of user data.

### PR DESCRIPTION
**Pull request by smn**
This makes use of that and removes vxpolls.urls which was a hacky way of
providing the same features but with messier data.

This requires praekelt/vxpolls#37 to land first.

### CODE REVIEW

File: go/urls.py [hunk 1]
```diff
@@ -37,7 +37,6 @@ def health(request):
         include('go.apps.urls')),
     url(r'^contacts/', include('go.contacts.urls', namespace='contacts')),
     url(r'^account/', include('go.account.urls', namespace='account')),
-    url(r'^', include('vxpolls.urls')),
```
Comment (dmaclay) [inline go/urls.py:40]: what was this line doing ?

**Comment by dmaclay** [comment]
looks ok :+1:

File: go/urls.py (hunk 1 above)
Comment (smn) [inline go/urls.py:40]: We were using the download views from vxpolls in the Go UI. This PR removes the need for that.

**Comment by smn** [comment]
This kills the `export_results` URL since the CSV it generated was pretty much useless and generated more questions than it was useful.

**Pull request merged**